src/components/Footer.tsx

```diff
@@ -0,0 +1,194 @@
+    margin-left: 0;
+    padding-bottom: 5rem;
+  }
+`;
+
+const Branding = styled.div`
+  ${atMinLg} {
+    flex: 1;
+  }
+`;
+
+const FooterLogo = styled(LogoIcon)`
+  display: block;
+  height: 3.5rem;
+  width: auto;
+
+  ${atMinMd} {
+    height: 3.5rem;
+  }
+`;
+
+const Slogan = styled.p`
+  color: ${({ theme }) => theme.colors.offWhite};
+  font-size: 1.875rem;
+  font-weight: 300;
+  line-height: 2.5rem;
+  letter-spacing: 0.06rem;
+  margin: 0.625rem 0 0;
+  width: 13.062rem;
+`;
+
+const Nav = styled.nav`
+  color: ${({ theme }) => theme.colors.offWhite};
+  font-size: 1.063rem;
+
+  ${atMinLg} {
+    flex: 1;
+  }
+
+  ul {
+    columns: 2;
+    list-style: none;
+    margin: 4rem 0 0 0;
+    padding-left: 0;
+
+    ${atMinMd} {
+      column-gap: 3rem;
+      margin-top: 0;
+    }
+
+    ${atMinLg} {
+      width: 60%;
+    }
+
+    li {
+      margin-bottom: 1.25rem;
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+
+      ${atMinMd} {
+        margin-bottom: 1.25rem;
+      }
+    }
+  }
+`;
+
+const ContactBlock = styled.address`
+  color: ${({ theme }) => theme.colors.offWhite};
+  font-size: 1.062rem;
+  line-height: 1.75rem;
+  letter-spacing: 0;
+  margin-top: ${({ theme }) => theme.spacing.xxl};
+  padding-bottom: ${({ theme }) => theme.spacing.lg};
+
+  ${atMinMd} {
+    margin-top: 0;
+    padding-bottom: 0;
+  }
+
+  ${atMinLg} {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    flex: 1;
+  }
+`;
+
+const ContactLink = styled(Link)`
+  display: block;
+  margin-bottom: ${({ theme }) => theme.spacing.md};
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &:first-of-type {
+    ${atMinLg} {
+      position: absolute;
+      right: 0;
+    }
+
+    ${atMinMd} {
+      margin-bottom: ${({ theme }) => theme.spacing.sm};
+    }
+  }
+
+  address {
+    font-style: normal;
+    margin: 2rem 0 0;
+
+    ${atMinMd} {
+      margin-top: 0;
+    }
+  }
+`;
+
+interface NavItemProps {
+  href: string;
+}
+const NavItem: FC<NavItemProps> = ({ href, ...props }) => {
+  return (
+    <li>
+      <NextLink href={href}>
+        <Link href={href} {...props} />
+      </NextLink>
+    </li>
+  );
+};
+
+export const Footer: FC = () => {
+  const theme = useTheme();
+  return (
+    <Container
+      as="footer"
+      css={css`
+        ${onlyXs} {
+          background-color: ${theme.colors.darkBlueAlt};
+        }
+      `}
+    >
+      <FooterWrapper>
+        <Branding>
+          <FooterLogo />
+          <Slogan>
+            Where code <br /> meets craft.
+          </Slogan>
+        </Branding>
+        <Nav>
+          <ul>
+            <NavItem href="/">Home</NavItem>
+            <NavItem href="/work">Work</NavItem>
+            <NavItem href="/approach">Approach</NavItem>
+            <NavItem href="/culture">Culture</NavItem>
+            <NavItem href="/careers">Careers</NavItem>
+            <NavItem href="/blog">Blog</NavItem>
+            <NavItem href="/contact">Contact</NavItem>
```

Comment:
💬 With the removal of the need for the `NextLink` this area becomes a little less redundant, and it may be more readable to remove the `NavItem` altogether. That's definitely at your discretion though.

```suggestion
            <li>
              <Link href="/">Home</Link>
            </li>
            <li>
              <Link href="/work">Work</Link>
            </li>
            <li>
              <Link href="/approach">Approach</Link>
            </li>
            <li>
              <Link href="/culture">Culture</Link>
            </li>
            <li>
              <Link href="/careers">Careers</Link>
            </li>
            <li>
              <Link href="/blog">Blog</Link>
            </li>
            <li>
              <Link href="/contact">Contact</Link>
            </li>
```